packages/labs/observers/src/intersection_controller.ts

```diff
@@ -7,7 +7,10 @@ import {
   ReactiveController,
   ReactiveControllerHost,
 } from '@lit/reactive-element/reactive-controller.js';
-
+import {
+  ObserverController,
+  type ObserverControllerConfig,
+} from './observer_controller.js';
```

Comment:
nit: I noticed a lot of other lit packages use kebab case instead of snake case for file naming. Is it worth changing the names of these controllers to normalize consumption with the remainder of the lit repo?